### PR TITLE
[MIG] agreement_maintenance: Migration to 13.0

### PR DESCRIPTION
**Pull request by olgamarcocb**
Standard Migration
cherry pick #761 
Depends on #760

### CODE REVIEW

File: agreement_maintenance/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    "name": "Maintenance Agreements",
+    "summary": "Manage maintenance agreements and contracts",
+    "author": "Pavlov Media, "
+    "Open Source Integrators, "
+    "Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/contract",
+    "category": "Maintenance",
+    "license": "AGPL-3",
+    "version": "13.0.1.0.0",
+    "depends": ["maintenance", "agreement_serviceprofile"],
```
Comment (ajaniszewska-dev) [inline agreement_maintenance/__manifest__.py:15]: travis has problem with agreement_serviceprofile module, probably requirements not properly updated?

Comment (etobella) [reply agreement_maintenance/__manifest__.py:15]: Well, this PR depends on the migration (#760)

File: agreement_maintenance/views/maintenance_equipment_view.xml [hunk 2]
```diff
@@ -0,0 +1,47 @@
+<!-- # Copyright (C) 2018 Pavlov Media
+     # Copyright (C) 2021 Open Source Integrators
+     # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record id="maintenance_equipment_form_view_agreement" model="ir.ui.view">
+        <field name="name">maintenance.equipment.form.agreement</field>
+        <field name="model">maintenance.equipment</field>
+        <field name="inherit_id" ref="maintenance.hr_equipment_view_form" />
```
Comment (albariera) [inline agreement_maintenance/views/maintenance_equipment_view.xml:8]: Could you add this, please?
A user without contract permission could not access the equipment.
 `<field name="groups_id" eval="[(4, ref('agreement_legal.group_agreement_readonly'))]" />`
Thanks! 
And if you could cherry-pick it to 14.0 :wink:

Comment (olgamarcocb) [reply agreement_maintenance/views/maintenance_equipment_view.xml:8]: Done! :smile: 
Cherry-pick: #799

Comment (albariera) [reply agreement_maintenance/views/maintenance_equipment_view.xml:8]: Perfect!

**Review by ajaniszewska-dev — APPROVED** [review]

**Review by etobella — APPROVED** [review]

**Review by albariera — APPROVED** [review]
LGTM :+1: :blush:

**Comment by rousseldenis** [comment]
@pedrobaeza Do you know if this comment is still applicable ?

image

**Comment by pedrobaeza** [comment]
I think it's not applicable, as the merge of agreement and contract didn't fructificate.

**Pull request merged**